packages/askar/src/wallet/AskarWallet.ts

```diff
@@ -116,8 +114,11 @@ export class AskarWallet extends AskarBaseWallet {
         keyMethod: askarWalletConfig.keyMethod,
         passKey: askarWalletConfig.passKey,
       })
+
+      // TODO: Should we do something to check if it exists?
+      // Like this.withSession()?
+
       this.walletConfig = walletConfig
-      this._session = await this._store.openSession()
     } catch (error) {
       // FIXME: Askar should throw a Duplicate error code, but is currently returning Encryption
```

Comment:
Just curious, do you know if this is still happening? There were quite a good number of fixes regarding this in Askar in the past months.

Reply:
No not sure, but we can find out :)